src/PerformanceTests/Tests/Categories/Base.cs

```diff
@@ -133,14 +133,4 @@ static void DeleteOtherPlatformHost(Permutation permutation)
             }
         }
     }
-
-    [SetUpFixture]
-    public class TestSessionInitializer
-    {
-        [SetUp]
```

Comment:
If it was style, I would not make the change. The previous code resulted in :

> OneTimeSetUp: SetUpAttribute attribute not allowed in a SetUpFixture

This resolved the run-time error.